scalability_and_performance/managing-bare-metal-hosts.adoc

```diff
@@ -0,0 +1,26 @@
+[id="managing-bare-metal-hosts"]
+= Managing bare metal hosts
+include::modules/common-attributes.adoc[]
+:context: managing-bare-metal-hosts
+
+toc::[]
+
+When you install {product-title} on a bare metal cluster, you can provision and manage bare metal nodes using `machine` and `machineset` custom resources (CRs) for bare metal hosts that exist in the cluster.
+
+include::modules/about-bare-metal-hosts-and-nodes.adoc[leveloffset=+1]
+include::modules/maintaining-bare-metal-hosts.adoc[leveloffset=+1]
+
+.Additional Resources
+
+* xref:../machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc[Adding compute machines to bare metal]
```

Comment:
xref:../machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc#adding-bare-metal-compute-user-infra[Adding compute machines to bare metal]